plasma_framework/contracts/mocks/exits/payment/routers/PaymentInFlightExitRouterMock.sol

```diff
@@ -19,11 +23,28 @@ contract PaymentInFlightExitRouterMock is PaymentInFlightExitRouter {
         inFlightExitMap.exits[exitId].exitMap = Bits.setBit(inFlightExitMap.exits[exitId].exitMap, 255);
     }
 
-    function getInFlightExitInput(uint192 exitId, uint8 inputIndex) public view returns (PaymentOutputModel.Output memory) {
+    function setInFlightExit(uint192 exitId, PaymentExitDataModel.InFlightExit memory exit) public {
+        PaymentExitDataModel.InFlightExit storage ife = inFlightExitMap.exits[exitId];
+        ife.exitStartTimestamp = exit.exitStartTimestamp;
+        ife.exitMap = exit.exitMap;
+        ife.position = exit.position;
+        ife.bondOwner = exit.bondOwner;
+        ife.oldestCompetitorPosition = exit.oldestCompetitorPosition;
+
+        for(uint i = 0 ; i < exit.inputs.length; i++) {
+            ife.inputs[i] = exit.inputs[i];
+        }
+
+        for(uint i = 0 ; i < exit.outputs.length; i++) {
+            ife.outputs[i] = exit.outputs[i];
+        }
+    }
+
+    function getInFlightExitInput(uint192 exitId, uint16 inputIndex) public view returns (PaymentExitDataModel.WithdrawData memory) {
         return inFlightExitMap.exits[exitId].inputs[inputIndex];
     }
 
-    function getInFlightExitOutput(uint192 exitId, uint8 outputIndex) public view returns (PaymentOutputModel.Output memory) {
+    function getInFlightExitOutput(uint192 exitId, uint16 outputIndex) public view returns (PaymentExitDataModel.WithdrawData memory) {
```

Comment:
`uint8 -> uint16` Why?

Reply:
we bump the general `outputIndex` from uint8 to uint16 from this issue: https://github.com/omisego/plasma-contracts/issues/168. As a result, I prefer uint16 in general when we are pointing to outputIndex.